packages/react-native-renderer/src/ReactNativeEventEmitter.js

```diff
@@ -100,6 +101,7 @@ function _receiveRootNodeIDEvent(
   batchedUpdates(function() {
     runExtractedPluginEventsInBatch(
       topLevelType,
+      RESPONDER_EVENT_SYSTEM,
```

Comment:
Why are we passing `RESPONDER_EVENT_SYSTEM`? That flag was meant for the new event system.

Reply:
😯🙃Nice catch! This was an oversight. It should be passing the plugin event system flag.